From: Azimjon Ulmasov <azimjon.ulmasov@chainguard.dev>
Date: Wed, 4 Feb 2026 18:34:00 +0000
Subject: [PATCH] Use local DocBook XSL stylesheets instead of downloading from internet

This patch replaces remote DocBook XSL stylesheet URLs with local file paths
to support hermetic container builds where network access is restricted.
---
 client/meson.build       | 2 +-
 client/pkcon/meson.build | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/client/meson.build b/client/meson.build
index 97ccdea66..946d60da4 100644
--- a/client/meson.build
+++ b/client/meson.build
@@ -50,7 +50,7 @@ if get_option('man_pages')
       xsltproc,
       '--nonet',
       '--output', '@OUTPUT@',
-      'http://docbook.sourceforge.net/release/xsl-ns/current/manpages/docbook.xsl',
+      '/usr/share/xml/docbook/xsl-ns-stylesheets/manpages/docbook.xsl',
       '@INPUT@',
     ],
     install: true,
diff --git a/client/pkcon/meson.build b/client/pkcon/meson.build
index fa2ac19cc..673fd7cce 100644
--- a/client/pkcon/meson.build
+++ b/client/pkcon/meson.build
@@ -33,7 +33,7 @@ if get_option('man_pages')
     command: [
       xsltproc,
       '--output', '@OUTPUT@',
-      'http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl',
+      '/usr/share/xml/docbook/xsl-ns-stylesheets/manpages/docbook.xsl',
       '@INPUT@',
     ],
     install: true,
@@ -46,7 +46,7 @@ if get_option('man_pages')
     command: [
       xsltproc,
       '--output', '@OUTPUT@',
-      'http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl',
+      '/usr/share/xml/docbook/xsl-ns-stylesheets/manpages/docbook.xsl',
       '@INPUT@',
     ],
     install: true,
-- 
2.52.0
